include/clangmetatool/collectors/find_calls_data.h

```diff
@@ -0,0 +1,63 @@
+#ifndef INCLUDED_CLANGMETATOOL_COLLECTORS_FIND_CALLS_DATA_H
+#define INCLUDED_CLANGMETATOOL_COLLECTORS_FIND_CALLS_DATA_H
+
+#include <clang/Frontend/CompilerInstance.h>
+#include <clang/ASTMatchers/ASTMatchers.h>
+
+namespace clangmetatool {
+  namespace collectors {
+
+    /**
+     * The data collected by the FindCalls collector
+     */
+    struct FindCallsData {
+
+      /**
+       * The declaration of the function and the context in which it occurs
+       */
+      std::multimap<const clang::FunctionDecl*,
+                    const clang::CallExpr*> call_context;
+
+      /**
+       * The reference to the function and the context in which it occurs
+       */
+      std::multimap<const clang::CallExpr*,
+                    const clang::DeclRefExpr*> call_ref;
```

Comment:
why is this a multimap?

Reply:
yes, actually this should be a map since we're only binding one function match in any context which case the key-values should be unique (unlike the above case where we can match the same function in different contexts)